flycheck-gometalinter.el

```diff
@@ -81,6 +81,14 @@ flycheck-gometalinter-enable-linters."
   duration."
   :safe #'stringp)
 
+(flycheck-def-option-var flycheck-gometalinter-concurrency "16" gometalinter
+  "Number of linters to run concurrently."
+  :safe #'stringp)
```

Comment:
Maybe something like this.

```
(flycheck-def-option-var flycheck-gometalinter-concurrency 16 gometalinter
  "Number of linters to run concurrently."
  :safe #'integerp)
```

and 

```
(option "--concurrency=" (number-to-string flycheck-gometalinter-concurrency) concat)
```

might work. Can you test, I'm in mobile right now.